MovieFirst/MovieFirst/Controllers/ListMoviesViewController.swift

```diff
@@ -0,0 +1,301 @@
+            popularButton.heightAnchor.constraint(equalToConstant: 40)
+        ])
+    }
+
+    private func createTopRatedButtonConstraint() {
+        topRatedButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            topRatedButton.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 5),
+            topRatedButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            topRatedButton.heightAnchor.constraint(equalToConstant: 40),
+            topRatedButton.leftAnchor.constraint(equalTo: popularButton.rightAnchor, constant: 10),
+        ])
+    }
+
+    private func createUpComingButtonConstraint() {
+        upComingButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            upComingButton.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 5),
+            upComingButton.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor),
+            upComingButton.widthAnchor.constraint(equalTo: topRatedButton.widthAnchor),
+            upComingButton.heightAnchor.constraint(equalToConstant: 40)
+        ])
+    }
+
+    private func createListMoviesTableViewConstraint() {
+        listMoviesTableView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            listMoviesTableView.topAnchor.constraint(equalTo: topRatedButton.bottomAnchor, constant: 5),
+            listMoviesTableView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor),
+            listMoviesTableView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor),
+            listMoviesTableView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor)
+        ])
+    }
+
+    private func createMainActivityIndicatorViewConstraint() {
+        mainActivityIndicatorView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            mainActivityIndicatorView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: -30),
+            mainActivityIndicatorView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor)
+        ])
+    }
+}
+
+// MARK: - UITableViewDelegate, UITableViewDataSource
+
+extension ListMoviesViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        movies?.count ?? 0
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard
+            let cell = tableView
+            .dequeueReusableCell(withIdentifier: Constants.movieTableViewCellText) as? MovieTableViewCell,
+            let safeMovies = movies
+        else { return UITableViewCell() }
+        cell.configureMovieTableViewCell(
+            dataImage: safeMovies[indexPath.row].dataImage,
```

Comment:
```suggestion
            movie: safeMovies[indexPath.row]
```
  передай структуру, а там уже нужные свойства установишь, зачем нам 5 параметров в методе, если хватит 1